Tests/UnitTests/SubscriberAttributes/SubscriberAttributesManagerTests.swift

```diff
@@ -1600,4 +1610,17 @@ private extension SubscriberAttributesManagerTests {
         expect(ipReceived.isSynced) == false
     }
 
+    func checkDeviceIdentifiersAreNotSet() {
+        let invokedParams = self.mockDeviceCache.invokedStoreParametersList
+
+        let idfvNilAttribute = invokedParams.first(where: { $0.attribute.key == "$idfv" })
+        expect(idfvNilAttribute) == nil
```

Comment:
`Nimble` doesn't support this, I think it's ignoring the expectation. For checking against `nil` you'd use `expect(x).to(beNil())`.

But I'd simply it since you're checking if it contains something, which would provide a much better failure message:
```swift
expect(invokedParams)
  .to(containElementSatisfying { $0.attribute.key == "$idfv" })
```

Reply:
Gotcha, that makes sense- to clarify, since I'm checking if it doesn't contain the attribute, this is how I updated it: 
```
expect(invokedParams).toNot(containElementSatisfying({ $0.attribute.key == "$idfv" }))
```

Does that make sense?

Reply:
Yup!